CONTRIBUTING.md

```diff
@@ -0,0 +1,56 @@
+# Contributing to rust-GameDig
+This project is very open to new suggestions, additions and/or changes, these 
+can come in the form of *discussions* about the project's state, *proposing a 
+new feature*, *holding a few points on why we shall do X breaking change* or
+*submitting a fix*.
+
+## Communications
+GitHub is the place we use to track bugs and discuss new features/changes,
+although we have a [Discord](https://discord.gg/NVCMn3tnxH) server for the
+community, all bugs, suggestions and changes will be reported on GitHub 
+alongside with their backing points to ensure the transparency of the project's
+development.
+
+## Issues
+When opening a new issue, make sure to fill the issue template. They are made
```

Comment:
Possibly worth mentioning that they should check whether there is an existing relevant issue first.